File: twosum.py
class twoSum:
    @staticmethod
    def getTwoSum(nums,target ):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        result = []
        left = 0
        right = len(nums)-1
        nums.sort()
        while left!=right:
            sum = nums[left]+nums[right]
            if sum==0:
                result += [nums[left],nums[right]]
                left +=1
            elif sum < nums[left]:
                left +=1
            else :
                right-=1
        return result
test =[2,1,0,0,-1,-2]
print(twoSum.getTwoSum(test,0))

File: twoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int [] result = new int[2];
        
        if(nums.length==0){
            return null;
        }
        for(int i =0;i< nums.length-1;i++){
            for(int j = i+1; j< nums.length;j++){
                if(nums[j]+nums[i]==target){
                    result[0] = i;
                    result [1] = j;
                    break;
                }
            }
        }
        
        return result;
        
    }
}